libs/live-config/src/providers/firebaseRemoteConfig/parser.ts

```diff
@@ -0,0 +1,32 @@
+import { Value } from "../firebaseRemoteConfig";
+import { ConfigInfo } from "../../LiveConfig";
+
+const objectParser = (value: unknown): object | undefined => {
+  const stringValue = (value as Value).asString();
+  try {
+    return stringValue ? JSON.parse(stringValue) : undefined;
+  } catch (error) {
+    console.error(`JSON.parse error: ${error}. Original string: ${stringValue}`);
+    return undefined;
+  }
+};
+
+export function parser(value: unknown, type: ConfigInfo["type"]) {
```

Comment:
Since you only use value as Value, you can make the assumption in the param type directly:
```suggestion
export function parser(value: Value, type: ConfigInfo["type"]) {
```

Reply:
fixed